flexpack/conan/conan_flexpack.go

```diff
@@ -0,0 +1,316 @@
+package conan
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/jfrog/build-info-go/entities"
+	"github.com/jfrog/build-info-go/flexpack"
+	"github.com/jfrog/gofrog/log"
+)
+
+// DependencyInfo is an alias for flexpack.DependencyInfo
+type DependencyInfo = flexpack.DependencyInfo
+
+// ConanFlexPack implements the FlexPackManager interface for Conan package manager.
+// It handles dependency resolution, checksum calculation, and build info collection.
+type ConanFlexPack struct {
+	config          ConanConfig
+	dependencies    []entities.Dependency
+	dependencyGraph map[string][]string
+	projectName     string
+	projectVersion  string
+	user            string
+	channel         string
+	conanfilePath   string
+	graphData       *ConanGraphOutput
+	requestedByMap  map[string][]string
+	initialized     bool
+}
+
+// NewConanFlexPack creates a new Conan FlexPack instance.
+// Note: Conanfile loading is deferred to first dependency parse for lazy initialization.
+func NewConanFlexPack(config ConanConfig) (*ConanFlexPack, error) {
+	cf := &ConanFlexPack{
+		config:          config,
+		dependencies:    []entities.Dependency{},
+		dependencyGraph: make(map[string][]string),
+		requestedByMap:  make(map[string][]string),
+	}
+
+	// Set default executable if not provided
+	if cf.config.ConanExecutable == "" {
+		cf.config.ConanExecutable = cf.getConanExecutablePath()
+	}
+
+	return cf, nil
+}
+
+// GetDependency returns dependency information as a formatted string
+func (cf *ConanFlexPack) GetDependency() string {
+	if err := cf.ensureInitialized(); err != nil {
+		log.Warn("Failed to initialize: " + err.Error())
+		return ""
+	}
+
+	var result strings.Builder
+	result.WriteString(fmt.Sprintf("Project: %s:%s\n", cf.projectName, cf.projectVersion))
+	result.WriteString("Dependencies:\n")
+
+	for _, dep := range cf.dependencies {
+		result.WriteString(fmt.Sprintf("  - %s [conan]\n", dep.Id))
+	}
+
+	return result.String()
+}
+
+// ParseDependencyToList converts parsed dependencies to a list format
+func (cf *ConanFlexPack) ParseDependencyToList() []string {
+	if err := cf.ensureInitialized(); err != nil {
+		log.Warn("Failed to initialize: " + err.Error())
+		return nil
+	}
+
+	var depList []string
+	for _, dep := range cf.dependencies {
+		depList = append(depList, dep.Id)
+	}
+
+	return depList
+}
+
+// CalculateScopes returns unique scopes from dependencies in standard order.
+// Conan supports the following scopes:
+//   - runtime: Regular dependencies (requires)
+//   - build: Build-time dependencies (build_requires, tool_requires)
+//   - test: Test dependencies (test_requires)
+//   - python: Python extension dependencies (python_requires)
+func (cf *ConanFlexPack) CalculateScopes() []string {
+	if err := cf.ensureInitialized(); err != nil {
+		return nil
+	}
+
+	scopesMap := make(map[string]bool)
+	for _, dep := range cf.dependencies {
+		for _, scope := range dep.Scopes {
+			scopesMap[scope] = true
```

Comment:
add the check here 

if scope is in []string{"runtime", "build", "test", "python"}
then go ahead and else don't add with a warn